shared/settings/chat/index.stories.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import * as React from 'react'
+import * as RPCChatTypes from '../../constants/types/rpc-chat-gen'
+import * as Sb from '../../stories/storybook'
+import Chat from '.'
+import {Box} from '../../common-adapters/index'
+
+const actions = {
+  onUnfurlSave: (mode: RPCChatTypes.UnfurlMode, whitelist: Array<string>) => {
+    Sb.action('onUnfurlSave')(mode, whitelist)
+  },
+  onRefresh: Sb.action('onRefresh'),
+}
+
+const props = {
+  unfurlMode: RPCChatTypes.unfurlUnfurlMode.whitelisted,
```

Comment:
I think we tend to find it unsatisfying when a display component's reaching all the way down into RPC definitions -- we sometimes duplicate the RPC shape inside `constants/settings` directly, or even re-export the RPC value directly from there.

Reply:
Yeah I thought this type was simple enough that it made sense here. I tried to make the unfurling types that come up to the UI as simple and usable as possible so it could just use them directly.